pkgs/os-specific/linux/powertop/default.nix

```diff
@@ -10,6 +10,10 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ gettext libnl ncurses pciutils pkgconfig zlib ];
 
+  patchPhase = ''
+    substituteInPlace src/main.cpp --replace "/sbin/modprobe" "/run/booted-system/sw/bin/modprobe"
```

Comment:
Well, at the very least, `module_init_tools` seems to be obsolete:
````
commit 4a83c97d3d3f376fa2a5ff886504aff4beb365d2
Author: Eelco Dolstra <eelco.dolstra@logicblox.com>
Date:   Wed Aug 7 18:30:30 2013 +0200

    Stage 1: Use modprobe from kmod instead of module-init-tools
    
    The latter is obsolete.
    
    http://hydra.nixos.org/build/5663799
````

But then, what's in `/run/booted-system/sw/bin/modprobe` is just a wrapper around `kmod`, so using it directly seems wrong too. Maybe the proper way could be a discussion topic for nix-dev.

Actually, in this case, modprobe is called via `system()` so just `modprobe` should work. I'll change it into that.